tests/manual-test-cases/Group5-Functional-Tests/5-19-ROBO-SKU.robot

```diff
@@ -15,60 +15,57 @@
 
-    Set Suite Variable  @{list}  ${esx1}  ${esx2}  ${esx3}  ${vc}
+    Log To Console  Set environment variables up for GOVC
+    Set Environment Variable  GOVC_URL  ${vc-ip}
+    Set Environment Variable  GOVC_USERNAME  Administrator@vsphere.local
+    Set Environment Variable  GOVC_PASSWORD  Admin\!23
+    Set Environment Variable  GOVC_INSECURE  1
 
-    Log To Console  Create a datacenter on the VC
-    ${out}=  Run  govc datacenter.create ha-datacenter
-    Should Be Empty  ${out}
-
-    Log To Console  Add ESX host to the VC
-    ${out}=  Run  govc host.add -hostname=${esx1-ip} -username=root -dc=ha-datacenter -password=e2eFunctionalTest -noverify=true
-    Should Contain  ${out}  OK
-    ${out}=  Run  govc host.add -hostname=${esx2-ip} -username=root -dc=ha-datacenter -password=e2eFunctionalTest -noverify=true
-    Should Contain  ${out}  OK
-    ${out}=  Run  govc host.add -hostname=${esx3-ip} -username=root -dc=ha-datacenter -password=e2eFunctionalTest -noverify=true
-    Should Contain  ${out}  OK
-
-    Create A Distributed Switch  ha-datacenter
-
-    Create Three Distributed Port Groups  ha-datacenter
-
-    Add Host To Distributed Switch  ${esx1-ip}
-    Add Host To Distributed Switch  ${esx2-ip}
-    Add Host To Distributed Switch  ${esx3-ip}
+    Add Host To Distributed Switch  /vcqaDC/host/cls
 
-    Add Vsphere License  %{ROBO_LICENSE}
-
-    # Assign license to each of the ESX servers
-    Assign Vsphere License  %{ROBO_LICENSE}  ${esx1-ip}
-    Assign Vsphere License  %{ROBO_LICENSE}  ${esx2-ip}
-    Assign Vsphere License  %{ROBO_LICENSE}  ${esx3-ip}
+    Log To Console  Enable HA
+    ${out}=  Run  govc cluster.change -ha-enabled cls
+    Should Be Empty  ${out}
 
-    Log To Console  Deploy VIC to the VC cluster
     Set Environment Variable  TEST_URL_ARRAY  ${vc-ip}
     Set Environment Variable  TEST_USERNAME  Administrator@vsphere.local
     Set Environment Variable  TEST_PASSWORD  Admin\!23
     Set Environment Variable  BRIDGE_NETWORK  bridge
     Set Environment Variable  PUBLIC_NETWORK  vm-network
-    Set Environment Variable  TEST_RESOURCE  /ha-datacenter/host/${esx1-ip}/Resources
-    Set Environment Variable  TEST_TIMEOUT  30m
+    Remove Environment Variable  TEST_DATACENTER
+    Set Environment Variable  TEST_DATASTORE  vsanDatastore
+    Set Environment Variable  TEST_RESOURCE  cls
+    Set Environment Variable  TEST_TIMEOUT  15m
+
+    # TODO - add ROBO license(s) to the test suite
+#    Add Vsphere License  %{ROBO_LICENSE}
+#
+#    :FOR  ${IDX}  IN RANGE  0  4
+#    \   Log To Console  Getting IP for ${user}-${name}.esx.${IDX}
+#    \   ${esx-ip}=  Get IP  ${user}-${name}.esx.${IDX}
+#    \   Log To Console  Applying ROBO license to host ${esx-ip}
+#    \   Assign Vsphere License  %{ROBO_LICENSE}  ${esx-ip}
```

Comment:
this is sorta the whole point of this no? why is it commented out?

Reply:
when you get the license you should just need to merge it to our internal repo in the vic-nightly-secrets.sh file and it will automatically get pulled into the next nightly run

Reply:
👍 that's the plan indeed.